README.md

```diff
@@ -257,6 +257,25 @@ Each tag object contains the following properties:
 
 
 
+### describe( [options], callback )
+
+Describes the currently checked out commit.
+
+Accepts most of the arguments that `git describe` accepts
+
+* `options` (Object): Options for `git describe` command
+  * `all` (Bool): adds '--all' if true.
+  * `tags` (Bool): adds '--tags' if true.
+  * `contains` (Bool): adds '--contains' if true.
+  * `debug` (Bool): adds '--debug' if true.
+  * `long` (Bool): adds '--long' if true.
+  * `always` (Bool): adds '--always' if true.
+  * `first_parent` (Bool): adds '--first-parent' if true.
```

Comment:
I wasn't exactly documenting all of git's options, I was listing the ones that was supported by the method.
Combined with the generic options code you proposed above it makes sense to take the approach you did for `Repo.clone()`, so I'll do that.